views/js/runner/plugins/tools/answerElimination/eliminator.js

```diff
@@ -113,6 +124,7 @@ define([
 
             //update plugin state based on changes
             testRunner
+                .on('loaditem', togglePlugin)
```

Comment:
You should also check `isEnabled` inside the `renderitem` handler (it does a `show` if a choice interaction is there)